inspire/version.py

```diff
@@ -31,11 +27,16 @@
 - dev has to be set in development mode (non-release).
 - revision can be set if you want to override the date coming from git.
 
-See the doctest below.
+See the tests.
 
 (This is taken from Invenio repository)
```

Comment:
I do not think this still exists anywhere.. it came from initial 2.0 demo overlay.